zap/src/main/java/org/parosproxy/paros/model/OptionsParam.java

```diff
@@ -92,7 +93,8 @@
     private Vector<AbstractParam> paramSetList = new Vector<>();
     private Map<Class<? extends AbstractParam>, AbstractParam> abstractParamsMap = new HashMap<>();
     private boolean gui = true;
-    private File userDirectory = null;
+    // ZAP: Change the default value of userDirectory from null.
```

Comment:
Move this to L44/45?

Reply:
+1 for moving to L44/45. The deprecation and code changes are part of another class, would it be okay to include those here?

Reply:
Not necessary.